### DISCUSSION TITLE
Ensure backward compatibility for OpenMP

### DISCUSSION THREAD

**Discussion by rmahjoubi** [pyccel/pyccel#1162] [Ideas]
**Description:**

Ensure backward compatibility for OpenMP until OpenMP 5.0, the user could use a flag to specify which version they are using

**Comment by EmilyBourne** [comment]
@rmahjoubi I have converted this to a discussion as it doesn't seem like you are sure of the right way to handle this yet. I think [this discussion](https://github.com/pyccel/pyccel/discussions/1159#discussioncomment-3169290) could also be relevant.

**Reply by EmilyBourne** [reply]
cc: @saidctb

**Comment by EmilyBourne** [comment]
I wonder if standard specifications should be part of the pyccel compiler description?

**Reply by saidctb** [reply]
Yes, we need to do that, I think the best way is to provide a default standard specification and give the user the possibility to choose other versions

**Reply by EmilyBourne** [reply]
By "part of the pyccel compiler description" I mean that it should be in the json file
